examples/demo-rollup/src/external_mock_rollup.rs

```diff
@@ -0,0 +1,168 @@
+    }
+
+    async fn create_endpoints(
+        &self,
+        state_update_receiver: StateUpdateReceiver<<Self::Spec as Spec>::Storage>,
+        sync_status_receiver: tokio::sync::watch::Receiver<SyncStatus>,
+        shutdown_receiver: tokio::sync::watch::Receiver<()>,
+        ledger_db: &LedgerDb,
+        sequencer: &SequencerCreationReceipt<Self::Spec>,
+        _da_service: &Self::DaService,
+        rollup_config: &RollupConfig<<Self::Spec as Spec>::Address, Self::DaService>,
+    ) -> anyhow::Result<NodeEndpoints> {
+        sov_modules_rollup_blueprint::register_endpoints::<Self, Native>(
+            state_update_receiver.clone(),
+            sync_status_receiver,
+            shutdown_receiver,
+            ledger_db,
+            sequencer,
+            rollup_config,
+        )
+        .await
+    }
+
+    async fn sequencer_additional_apis<Seq>(
+        &self,
+        sequencer: Arc<Seq>,
+        rollup_config: &RollupConfig<<Self::Spec as Spec>::Address, Self::DaService>,
+    ) -> anyhow::Result<NodeEndpoints>
+    where
+        Seq: Sequencer<Spec = Self::Spec, Rt = Self::Runtime, Da = Self::DaService>,
+    {
+        let eth_signer = eth_dev_signer();
+        let eth_rpc_config = EthRpcConfig {
+            eth_signer,
+            extension: rollup_config.extension_or_panic(),
+            buffer_raw_txs: true,
+        };
+
+        Ok(NodeEndpoints {
+            jsonrpsee_module: sov_ethereum::get_ethereum_rpc(eth_rpc_config, sequencer)
+                .remove_context(),
+            ..Default::default()
+        })
+    }
+
+    async fn create_da_service(
+        &self,
+        rollup_config: &RollupConfig<<Self::Spec as Spec>::Address, Self::DaService>,
+        _shutdown_receiver: tokio::sync::watch::Receiver<()>,
+    ) -> Self::DaService {
+        StorableMockDaClient::from_config(rollup_config.da.clone())
+            .expect("Failed to create da service")
+    }
+
+    async fn create_prover_service(
+        &self,
+        prover_config: RollupProverConfig<Risc0>,
+        rollup_config: &RollupConfig<<Self::Spec as Spec>::Address, Self::DaService>,
+        _da_service: &Self::DaService,
+    ) -> Self::ProverService {
+        let (host_args, prover_config_discriminant) = prover_config.split();
+        let inner_vm = Risc0Host::new(*host_args);
+
+        let outer_vm = MockZkvmHost::new_non_blocking();
+        let da_verifier = Default::default();
+
+        ParallelProverService::new_with_default_workers(
+            inner_vm,
+            outer_vm,
+            da_verifier,
+            prover_config_discriminant,
+            CodeCommitment::default(),
+            rollup_config.proof_manager.prover_address,
+        )
+    }
+
+    fn create_storage_manager(
+        &self,
+        rollup_config: &RollupConfig<<Self::Spec as Spec>::Address, Self::DaService>,
+    ) -> anyhow::Result<Self::StorageManager> {
+        NativeStorageManager::new(&rollup_config.storage.path)
```

Comment:
Optional for this PR: NOMT is current recommeneded default, so maybe we want to test with NomtStorageManager, to be closer to production setup.